### PR TITLE
Scrapper, Polina Tokunova - 22FPL1

### PR DESCRIPTION
**Pull request by polsha24**

### CODE REVIEW

**Comment by dmitry-uraev** [comment]
Good progress. Please, continue  implementing lab and switch to parser.

File: lab_5_scrapper/scrapper.py [hunk 1]
```diff
@@ -127,11 +229,31 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
...
         """
+        seed_urls = self.get_search_urls()
+
+        for seed_url in seed_urls[:1]:
+            response = make_request(seed_url, self.config)
+            if not response.ok:
+                continue
+
+            article_bs = BeautifulSoup(response.text, "html.parser")
+            urls = [self._extract_url(article_bs) for _ in range(10)]
```
Comment (dmitry-uraev) [inline lab_5_scrapper/scrapper.py:255]: setting your loop to exact range 10 here is not safe

are you sure this number of iterations is enough in all crawler uasage cases? e.g. i would like to crawl 100 articles?

Comment (dmitry-uraev) [reply lab_5_scrapper/scrapper.py:255]: what about using while?

**Comment by dmitry-uraev** [comment]
You have Read timed out error - try changing request parameters to pass tests

File: lab_5_scrapper/scrapper.py (hunk 1 above)
Comment (polsha24) [reply lab_5_scrapper/scrapper.py]: done

**Pull request closed without merging**